xbmc/guilib/GUIBaseContainer.cpp

```diff
@@ -68,9 +68,9 @@ CGUIBaseContainer::~CGUIBaseContainer(void)
   delete m_listProvider;
 }
 
-void CGUIBaseContainer::DoProcess(unsigned int currentTime, CDirtyRegionList &dirtyregions)
+void CGUIBaseContainer::DoProcess(CGUIRenderInfo &renderInfo)
```

Comment:
Just a question: why are all the new params not <code>const CGUIRenderInfo &</code>? Is render info supposed to be altered by that methods? If not, I would go for const wehenever possible.